leasing/models/rent.py

```diff
@@ -241,6 +300,24 @@ class Rent(TimeStampedSafeDeleteModel):
         null=True,
     )
 
+    # In Finnish: Tasotarkistusindeksi
+    old_dwellings_in_housing_companies_price_index = models.ForeignKey(
+        OldDwellingsInHousingCompaniesPriceIndex,
+        verbose_name=_("Old dwellings in housing companies price index"),
+        related_name="+",
+        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+    )
+
+    # In Finnish: Tasotarkistusindeksin pisteluku vuokran alkaessa (edellisen vuoden keskiarvo)
+    start_price_index_point_figure = models.DecimalField(
+        verbose_name=_("Start price index point figure"),
+        decimal_places=1,
+        max_digits=8,
+        null=True,
+    )
+
```

Comment:
Then I think we need a 3rd new property: type of the tasotarkistus (20y/20y or 20y/10y).

The Rent class is getting slightly populated, but it might still be OK because I just removed 4 fields about the seasonal rent.

Another possibility is to create a new model for `PeriodicRentAdjustment` which holds these 3 fields (index, adjustment type, and starting point figure). Then Rent would have a single ForeignKey property like `periodic_rent_adjustment` that can be null if the rent doesn't use the feature.

Reply:
What do you @henrinie-nc  think?